<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="comboWidget">
   <widget class="QGroupBox" name="signinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Sign In</string>
    </property>
    <widget class="QPushButton" name="signinSubmit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>164</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="comboVLayout">
      <item>
       <layout class="QHBoxLayout" name="emailHLayout">
        <item>
         <widget class="QLabel" name="emailLabel">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailInput"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="passwordHLayout">
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordInput">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="loginErrorLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>293</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogin">
    <property name="title">
     <string>Login</string>
    </property>
   </widget>
   <addaction name="menuLogin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
